### ISSUE TITLE
Create thumbnails without adding suffix

### ISSUE THREAD

**Issue by Bokenbergen** [xuminic/ezthumb#8]
Hi,

I want to batch thumbnails without adding any suffixes. If i don't use the --suffix parameter it will add _thumb. How do i achieve this?
-x,--suffix ARGS         the suffix of output filename (_thumb)

**Comment by AdKiller** [comment]
this program has not been updated since 2017...you will have better luck just using a batch renaming utility.

**Comment by Bokenbergen** [comment]
> this program has not been updated since 2017...you will have better luck just using a batch renaming utility.

Yes I know, and i could do the renaming with PS. However since I run my batches continuously they would re-create all the files since they are no longer the correct suffix. :)
